templates/chat/item/item-description.hbs

```diff
@@ -4,419 +4,14 @@
         <h4>{{item.name}}</h4>
     </header>
     <div class="chat-description">
-        {{#if (or (eq type 'armor') (eq type 'weapon') (eq type 'diagrams') (eq type 'mount') (eq type 'valuable'))}}
-            <div class="list-item-description">
-                {{#if item.system.description}}
-                    <h4>{{localize "WITCHER.Item.Description"}}</h4>
-                    <span>{{item.system.description}}</span>
-                {{/if}}
-            </div>
-        {{/if}}
-        {{#if (or (eq type 'spell') (eq type 'hex') (eq type 'ritual'))}}
-            <div class="list-item-description">
-                {{#if item.system.effect}}
-                    <h4>{{localize "WITCHER.Item.Effect"}}</h4>
-                    <span>{{item.system.effect}}</span>
-                {{/if}}
-            </div>
-            <div class="list-item-side-effect">
-                {{#if item.system.sideEffect}}
-                    <h4>{{localize "WITCHER.Spell.SideEffect"}}</h4>
-                    <span>{{item.system.sideEffect}}</span>
-                {{/if}}
-            </div>
-            <div class="list-item-requirements">
-                {{#if item.system.liftRequirement}}
-                    <h4>{{localize "WITCHER.Spell.Requirements"}}</h4>
-                    <span>{{item.system.liftRequirement}}</span>
-                {{/if}}
-            </div>
-        {{/if}}
-        {{#if (or (eq type 'alchemical') (eq type 'mutagen'))}}
-            {{#if item.system.effect}}
-                <div class="list-item-description">
-                    <h4>{{localize "WITCHER.Item.Description"}}</h4>
-                    <span>{{#if (eq ../itemType 'mutagen')}}<b>{{localize "WITCHER.Item.Effect"}}: </b>{{/if}} {{alchemical.system.effect}}</span>
-                </div>
-            {{/if}}
-            {{#if item.system.minorMutation}}
-                <span><b>{{localize "WITCHER.Mutagen.MinorMutation"}}: </b>{{item.system.minorMutation}}</span>
-            {{/if}}
-        {{/if}}
-        {{#if (eq type 'component')}}
-            <div class="list-item-description">
-                <h4>{{localize "WITCHER.Item.Description"}}</h4>
-                <span><b>{{localize "WITCHER.Component.Location"}}:</b> {{item.system.location}}</span>
-            </div>
-        {{/if}}
+        {{> "systems/TheWitcherTRPG/templates/chat/item/partials/item-description/description.hbs" type=type description=item.system.description }}

```

Comment:
If you only need item info you dont need to have them as explicit variables. If you provide none the item will be passed through